android/capacitor/src/main/java/com/getcapacitor/Plugin.java

```diff
@@ -184,7 +184,7 @@ public PluginCall getSavedCall() {
      * @return The saved plugin call
      */
     protected PluginCall getPermissionCall() {
```

Comment:
We should document that this dequeues the call ID now. Since this is no longer idempotent, a rename may be warranted as well.